internal/types/studies.go

```diff
@@ -58,23 +58,23 @@ type StudyAdmin struct {
 
 type Asset struct {
 	ModelAuditable
-	StudyID                uuid.UUID `gorm:"not null;index"`
-	Title                  string    `gorm:"not null"`
-	Description            string    `gorm:"type:text;not null"`
-	ClassificationImpact   string    `gorm:"not null"`
-	Protection             string    `gorm:"not null"`
-	LegalBasis             string    `gorm:"not null"`
-	Format                 string    `gorm:"not null"`
-	Expiry                 string    `gorm:"not null"`
-	HasDspt                bool      `gorm:"not null;default:false"`
-	StoredOutsideUkEea     bool      `gorm:"not null;default:false"`
-	AccessedByThirdParties bool      `gorm:"not null;default:false"`
-	ThirdPartyAgreement    string    `gorm:""`
-	Status                 string    `gorm:"not null"`
+	CreatorUserID        uuid.UUID `gorm:"not null;index"`
+	StudyID              uuid.UUID `gorm:"not null;index"`
+	Title                string    `gorm:"not null"`
+	Description          string    `gorm:"type:text;not null"`
+	ClassificationImpact string    `gorm:"not null"`
+	Protection           string    `gorm:"not null"`
+	LegalBasis           string    `gorm:"not null"`
+	Format               string    `gorm:"not null"`
+	Expiry               string    `gorm:"not null"`
```

Comment:
I think this should be a `time.Time` and called `ExpiresAt` (will need to be careful about the migration as gorm doesn't drop columns automatically)